config/config.php

```diff
@@ -181,6 +181,19 @@
 
     'global_id_field' => 'id',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Global ID resolver
+    |--------------------------------------------------------------------------
+    |
+    | The Global ID resolver class.
+    | You can set your own here, as long as it implements the following
+    | interface: \Nuwave\Lighthouse\Support\Contracts\GlobalId
+    |
+    */
+
+    'global_id_resolver' => \Nuwave\Lighthouse\Execution\Utils\GlobalId::class,
```

Comment:
We can just let the user rebind this via the container themselves and keep the config slim.